src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents the association class between Person and Tag.
+ * Each object represents an adding or deleting of a tag for a specific person
+ * that happened in the session.
+ */
+public class Tagging {
+	
+	private final static String PREFIX_ADDITION = "+";
+	private final static String PREFIX_DELETION = "-";
+	private final static String SEPARATOR = " ";
+	
+	private boolean isAdd;
+	private Tag tag;
+	private Person person;
+	
+	public Tagging (Person person, Tag tag, boolean isAdd) {
+		this.person = person;
+		this.tag = tag;
+		this.isAdd = isAdd;
+	}
+	
+	public String toString() {
+		if (isAdd) {
+			return PREFIX_ADDITION + SEPARATOR + person.getName() + tag.toString();
+		}
+		else
```

Comment:
multiple coding standard violations here.
- else should be cuddled between braces
- single line conditions should also be enclosed in braces
- indentation is done using tabs and not 4 space.